PersonBook/PersonBook/PersonBook/Service/PersonService.cs

```diff
@@ -0,0 +1,68 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace PersonBook.Service
+{
+    using CustomExceptions;
+    using Entity;
+
+    public class PersonService : IPersonService
+    {
+        private ICollection<Person> _personList;
+        public PersonService()
+        {
+            _personList = new List<Person>();
+        }
+
+        public void AddPerson(string name, string surname)
+        {
+            int Id = IdGenerator();
```

Comment:
variable names should start with lowercase.
`var id = IdGenerator();`

Reply:
Method names should start with verbs.
That should be `GenerateId()`

Reply:
Done